src/EditorFeatures/CSharpTest/CodeActions/AddAwait/AddAwaitTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.CodeRefactorings.AddAwait;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings.AddAwait
+{
+    [Trait(Traits.Feature, Traits.Features.AddAwait)]
+    public class AddAwaitTests : AbstractCSharpCodeActionTest
+    {
+        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
+            => new CSharpAddAwaitCodeRefactoringProvider();
+
+        [Fact]
+        public async Task Simple()
+        {
+            await TestInRegularAndScriptAsync(@"
+using System.Threading.Tasks;
+class Program
+{
+    async Task<int> GetNumberAsync()
+    {
+        var x = GetNumberAsync()[||];
+    }
+}", @"
+using System.Threading.Tasks;
+class Program
+{
+    async Task<int> GetNumberAsync()
+    {
+        var x = await GetNumberAsync();
```

Comment:
can you have another option which also adds `.ConfigureAwait(false)`?

Reply:
Rebased so that I could test on 15.9 and added a second fix option.

![image](https://user-images.githubusercontent.com/12466233/45197797-02c19380-b218-11e8-898f-997393ba3f33.png)

Reply:
thanks!